html/modules/custom/docstore/src/Commands/DocstoreCommands.php

```diff
@@ -137,6 +149,785 @@ public function __construct(
+      // Glide.
+      if (isset($row->fields->glide)) {
+        $node->set('glide', $row->fields->glide);
+      }
+
+      // Profile.
+      if (isset($row->fields->profile->overview)) {
+        $node->set('profile', $row->fields->profile->overview);
+      }
+
+      // Description.
+      if (isset($row->fields->description)) {
+        $node->set('description', $row->fields->description);
+      }
+
+      // Disaster type.
+      if (isset($row->fields->type) && !empty($row->fields->type)) {
+        $type_data = [];
+        foreach ($row->fields->type as $type) {
+          $query = $this->entityTypeManager
+            ->getStorage('taxonomy_term')
+            ->getQuery()
+            ->condition('vid', 'disaster_types')
+            ->condition('common_disaster_type_code', $type->code);
+          $term_ids = $query->execute();
+          $term_id = reset($term_ids);
+          if (!empty($term_id)) {
+            $type_data[] = [
+              'target_id' => $term_id,
+            ];
+          }
+        }
+        $node->set('disaster_type', $type_data);
+      }
+
+      // Primary disaster type.
+      if (isset($row->fields->primary_type) && !empty($row->fields->primary_type)) {
+        $query = $this->entityTypeManager
+          ->getStorage('taxonomy_term')
+          ->getQuery()
+          ->condition('vid', 'disaster_types')
+          ->condition('common_disaster_type_code', $row->fields->primary_type->code);
+        $term_ids = $query->execute();
+        $term_id = reset($term_ids);
+        if (!empty($term_id)) {
+          $type = ['target_id' => $term_id];
+          $node->set('primary_disaster_type', $type);
+        }
+      }
+
+      // Countries.
+      if (isset($row->fields->country) && !empty($row->fields->country)) {
+        $country_data = [];
+        foreach ($row->fields->country as $country) {
+          $query = $this->entityTypeManager
+            ->getStorage('taxonomy_term')
+            ->getQuery()
+            ->condition('vid', 'countries')
+            ->condition('common_iso3', $country->iso3);
+          $term_ids = $query->execute();
+          $term_id = reset($term_ids);
+          if (!empty($term_id)) {
+            $country_data[] = [
+              'target_id' => $term_id,
+            ];
+          }
+        }
+        $node->set('countries', $country_data);
+      }
+
+      // Primary country.
+      if (isset($row->fields->primary_country) && !empty($row->fields->primary_country)) {
+        $query = $this->entityTypeManager
+          ->getStorage('taxonomy_term')
+          ->getQuery()
+          ->condition('vid', 'countries')
+          ->condition('common_iso3', $row->fields->primary_country->iso3);
+        $term_ids = $query->execute();
+        $term_id = reset($term_ids);
+        if (!empty($term_id)) {
+          $country = ['target_id' => $term_id];
+          $node->set('primary_country', $country);
+        }
+      }
+
+      $node->set('author', 'RW');
```

Comment:
Is this right? It came from `create_disasters.php` but elsewhere we have 'author' as 'Shared'.

Reply:
`Shared` is probably better